src/Tribe/Admin/Notice/Service_Provider.php

```diff
@@ -54,24 +52,5 @@ public function plugins_loaded() {
 		tribe( 'pue.notices' );
 		tribe( 'admin.notice.php.version' );
 		tribe( WP_Version::class );
-
-		if ( defined( 'TRIBE_HIDE_MARKETING_NOTICES' ) ) {
-			return;
-		}
```

Comment:
It would still be nice to allow people to choose not to see those IMO

Reply:
Oh yeah they will be Able to do that I am going to Extend the `should_display()` and use the correct method `tec_should_hide_upsell()`

Reply:
Added